external-crates/move/crates/move-compiler/src/naming/ast.rs

```diff
@@ -825,6 +830,11 @@ impl TypeName_ {
     }
 }
 
+pub static UNIT_TYPE: Lazy<Type_> = Lazy::new(|| TypeInner::Unit.into());
+pub static ANYTHING_TYPE: Lazy<Type_> = Lazy::new(|| TypeInner::Anything.into());
+pub static VOID_TYPE: Lazy<Type_> = Lazy::new(|| TypeInner::Void.into());
+pub static UNRESOLVED_ERROR_TYPE: Lazy<Type_> = Lazy::new(|| TypeInner::UnresolvedError.into());
+
```

Comment:
Doesn't have to be this PR, but this was used before the oncecell/lock and lazycell/lock were standardized. We might want to switch to those

Reply:
I will do this as a follow-up PR to remove `Lazy` everywhere,